internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -346,6 +350,29 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 		}
 	}
 
+	supportAppsWithDaemonSets := map[string]string{"calico-node": "kube-system"}
+	for daemonSet, namespace := range supportAppsWithDaemonSets {
+
+		pods, err := k8sClient.GetPodsFromDaemonSet(namespace, daemonSet)
+		if err != nil {
+			return err
+		}
+		if len(pods.Items) == 0 {
+			return fmt.Errorf("no pods found from %q/%q daemonSet", namespace, daemonSet)
```

Comment:
nit: this may make the logs look a bit cleaner
```suggestion
			return fmt.Errorf("no pods found from %s/%s daemonSet", namespace, daemonSet)
```

Reply:
done